Tools/Scripts/configure-xcode-for-embedded-development

```diff
@@ -227,7 +227,11 @@ def do_copy(source_path, dest_path):
         if dest_path.exists():
             shutil.rmtree(dest_path)
```

Comment:
A little surprised that we don't need to change anything here.

Reply:
This is mostly for when you're re-running the script...I suppose you could capture the whole thing